PagerDuty/pagerduty.30s.rb

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/ruby
+# <bitbar.title>PagerDuty</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.a uthor>Allan Frese</bitbar.author>
+# <bitbar.author.github>frese</bitbar.author.github>
+# <bitbar.desc>Shows current pagerduty alert status.</bitbar.desc>
+# <bitbar.dependencies>ruby (httparty gem)</bitbar.dependencies>
```

Comment:
It'd be awesome to see a screenshot of the plugin, linked with the `<bitbar.image>` metadata.